### ISSUE TITLE
Positron on `conda-forge`

### ISSUE THREAD

**Issue by roaldarbol** [posit-dev/positron#3767]
It would be great to get Positron on conda-forge ([similar to RStudio](https://github.com/conda-forge/rstudio-feedstock)), as it would allow a global (or project-specific) installation with Pixi.

**Comment by wesm** [comment]
It looks like the RStudio conda-forge recipe is community maintained, and building Positron is (I think!) easier than RStudio, so hopefully this will not be a heavy lift for a motivated early user.

**Comment by roaldarbol** [comment]
Sounds good! I'm not really knowledgeable about building and recipes, but I'll try rallying someone. :-)

**Comment by wesm** [comment]
See also #3725. Since Positron wants to bundle Ark with it, we might consider making improvements to the Positron build system to depend on an installed version of Ark in the environment (so it isn't necessary to bundle, and Positron can declare a dependency on Ark in its conda recipe)

**Comment by roaldarbol** [comment]
Yeah it would be great to have both on conda-forge (and also along the lines of why I opened both issues). IMO Ark would be more important to get on as we could use that elsewhere for Jupyter and VS Code setups.

**Comment by matthewfeickert** [comment]
As ARK now exists on conda-forge (https://github.com/conda-forge/ark-feedstock (c.f. https://github.com/posit-dev/positron/issues/3725)) is there any known technical blocker for this? Or is it hypothetically feasible to attempt a PR to https://github.com/conda-forge/staged-recipes/?

**Comment by matthewfeickert** [comment]
Cross-reference with https://github.com/conda-forge/staged-recipes/issues/32424
